README.rst

```diff
@@ -70,9 +70,74 @@ We appreciate any help!
 BUILD & DEVELOPMENT
 -------------------
 
-Run ``python3 setup.py install``
+1. Clone the *coala-html* repository
 
-Note: You need to have bower installed before you may install the python packages.
+::
+
+  $ git clone https://github.com/coala/coala-html.git
+
+2. Install latest version of **Node.js**
+
+- *Configure Node.js PPA*
+
+::
+
+    $ sudo apt-get install curl python-software-properties
+    $ curl -sL https://deb.nodesource.com/setup_11.x | sudo bash -
+
+- *Install Node.js and npm*
+
+::
+
+  $ sudo apt-get install nodejs
```

Comment:
I think option 2 will be a better option, as if we add commands for other distributions then it will take more space.
So should I add only the link and remove the existing installation instructions for ubuntu or add an extra line that mentions the link of insallation instruction for other distributions ?

Reply:
Latter would be fine imo